.github/workflows/ci-testing.yml

```diff
@@ -195,8 +195,8 @@ jobs:
             done
 
             #set targets to just sycl iterator for dpcpp tests
-            make_targets="build-onedpl-sycl_iterator-tests build-onedpl-ranges-tests"
-            ctest_flags="-R (sycl_iterator_.*)|(std_ranges_.*)\.pass"
+            make_targets="build-onedpl-sycl_iterator-tests build-onedpl-ranges-tests build-onedpl-implementation_details-tests"
```

Comment:
With this change, we only compile and run the device copyable tests on Linux. Do we want to add a similar change for Windows CI?

Reply:
Alright, after a few attempts, windows looks like its working.